tools/xcodeproj_shims/installers/_swiftmodule.sh

```diff
@@ -2,18 +2,31 @@
 
 set -euo pipefail
 
-for module in "$@"; do
-    doc="${module%.swiftmodule}.swiftdoc"
-    module_name=$(basename "$module")
-    module_bundle="$BUILT_PRODUCTS_DIR/$module_name"
-    mkdir -p "$module_bundle"
+# Xcode 12 and under stored index information in /DerivedData/<project>/Build/Products
+readonly xcode_12_index_dir="$BUILT_PRODUCTS_DIR"
 
-    cp "$module" "$module_bundle/$CURRENT_ARCH.swiftmodule"
-    cp "$doc" "$module_bundle/$CURRENT_ARCH.swiftdoc"
+# Xcode 13 and later store index information in /DerivedData/<project>/Index/Build/Products
+readonly xcode_13_index_dir="${BUILT_PRODUCTS_DIR/\/Build\/Products\///Index/Build/Products/}"
 
-    ios_module_name="$CURRENT_ARCH-$LLVM_TARGET_TRIPLE_VENDOR-$SWIFT_PLATFORM_TARGET_PREFIX$LLVM_TARGET_TRIPLE_SUFFIX"
-    cp "$module" "$module_bundle/$ios_module_name.swiftmodule"
-    cp "$doc" "$module_bundle/$ios_module_name.swiftdoc"
+readonly dirs=(
+  "$xcode_12_index_dir"
+  "$xcode_13_index_dir"
+)
 
-    chmod -R +w "$module_bundle"
+for index_dir in "${dirs[@]}"; do
+    for module in "$@"; do
+        doc="${module%.swiftmodule}.swiftdoc"
+        module_name=$(basename "$module")
+        module_bundle="$index_dir/$module_name"
+        mkdir -p "$module_bundle"
+
+        cp "$module" "$module_bundle/$CURRENT_ARCH.swiftmodule" || true
+        cp "$doc" "$module_bundle/$CURRENT_ARCH.swiftdoc" || true
```

Comment:
The `|| true` is an unrelated change I tacked on after observing this script can fail early, and thwart swiftmodule copying, if a file does not exist (something I observed for `swiftdoc` specifically)

This isn't meant to mask over another problem, just prevent total failure as stderr logs will still reflect any problems copying.